src/smart-components/request/expandable-content.js

```diff
@@ -4,7 +4,15 @@ import { Link } from 'react-router-dom';
 import { TextContent, Text, TextVariants, Level, LevelItem, Button } from '@patternfly/react-core';
 import { isRequestStateActive } from '../../helpers/shared/helpers';
 
-const ExpandableContent = ({ id, content, state }) => {
+const expandedItem = (title, detail) => {
```

Comment:
I think we be using it as a component not as a pure function. Its a bit confisung

```jsx
const ExpandedItem = ({title, detail}) => ...
```